src/main/java/ro/chirila/programarispital/repository/dto/AppointmentRequestDTO.java

```diff
@@ -0,0 +1,105 @@
+package ro.chirila.programarispital.repository.dto;
+
+import ro.chirila.programarispital.repository.entity.PeriodOfAppointment;
+import ro.chirila.programarispital.repository.entity.TypeOfService;
+import ro.chirila.programarispital.repository.entity.User;
+
+import java.time.LocalTime;
+import java.util.Date;
+import java.util.List;
+
+public class AppointmentRequestDTO {
```

Comment:
I recommend using the `record` classes from java. They are created especially for DTOs

Reply:
Kinda fixed, didn't understand how to access my setter for AppointmentResponseDTO